pkg/controller/hyperconverged/hyperconverged_controller.go

```diff
@@ -30,6 +32,7 @@ var (
 	KubeVirtImagePullPolicy      = "IfNotPresent"
 	CDIImagePullPolicy           = "IfNotPresent"
 	NetworkAddonsImagePullPolicy = "IfNotPresent"
```

Comment:
The vars are going to be removed, so need to add and `ImagePullPolicy`

Reply:
Not sure I understood, could you please rephrase? keeping the hunk temporarily.

Reply:
The HCO is not going to expose any component configuration right now.  We're removing it in #40.  So remove `SSPImagePullPolicy` and any code that uses it.

Reply:
ack.